qiskit_optimization/translators/utils.py

```diff
@@ -0,0 +1,48 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Utilities for optimization model translators"""
+
+from typing import Any, TYPE_CHECKING
+
+from qiskit_optimization.exceptions import QiskitOptimizationError
+from .docplex import DocplexMpTranslator
+
+if TYPE_CHECKING:
+    # pylint: disable=cyclic-import
+    from qiskit_optimization.problems.quadratic_program import QuadraticProgram
+
+translators = [DocplexMpTranslator()]
```

Comment:
Well, I mean to have something like this:
```python
translators = {Model: DocplexMpTranslator()}

# create a DOcplex model
some_model = Model()

# load the model
qp = translators[type(some_model)].model_to_qp(model)
```
This would just remove the loop.

Reply:
Oh, I see, then maybe it is easier to keep the list approach!